src/client/styles/components/Feedback.scss

```diff
@@ -0,0 +1,83 @@
+.feedback {
+  position: fixed;
+  max-width: 400px;
+  bottom: 0;
+  right: 0;
+  z-index: 100;
+  text-align: right;
+}
+
+.feedback-button {
```

Comment:
Just curious, any reason why this class is not nested under the `.feedback` class above?